src/kyc.ts

```diff
@@ -56,10 +66,40 @@ export type PersonalDataAndDocumentsKyc = z.infer<
   typeof personalDataAndDocumentsKycSchema
 >
 
+export const personalDataAndDocumentsDetailedKycSchema = z.object({
+  firstName: z.string(),
+  middleName: z.string().optional(),
+  lastName: z.string(),
+  dateOfBirth: z.object({
+    day: z.string(),
+    month: z.string(),
+    year: z.string(),
+  }),
+  address: z.object({
+    address1: z.string(),
+    address2: z.string().optional(),
+    isoCountryCode: z.string(),
+    isoRegionCode: z.string(),
+    city: z.string(),
+    postalCode: z.string().optional(),
+  }),
+  phoneNumber: z.string(),
+  email: z.string(),
+  selfieDocument: z.string(),
+  identificationDocumentType: z.enum(['IDC', 'PAS', 'DL']),
```

Comment:
Please reference the type created above rather than repeating the hardcoded string values here.